README.md

```diff
@@ -1,3 +1,7 @@
 # Kartoffelskript Language
 
-Copyright 2018 Project Kartoffelskript.
+Kartoffelskript is a programming language with potatoes.
```

Comment:
Why putting jokes into the README? The language being is esoteric should not mean, that it can't be implemented in a professional manner.